package.json

```diff
@@ -37,18 +37,18 @@
   },
   "homepage": "https://github.com/sugarandmagic/sequelize-mv-support#readme",
   "dependencies": {
-    "pg": "^7.17.1",
+    "pg": "^7.18.1",
     "sequelize": "^6.0.0-beta.4"
   },
   "engines": {
-    "node": ">=10.16.0"
+    "node": ">=12.14.1"
```

Comment:
this should really match the output of the compiled stuff: https://github.com/sugarandmagic/sequelize-mv-support/blob/master/.babelrc#L8
otherwise people running node 10 won't be able to install even though the compiled module would run fine for them.